lib/mix/tasks/conduit.gen.broker.ex

```diff
@@ -204,6 +204,19 @@ defmodule Mix.Tasks.Conduit.Gen.Broker do
   <% end %>
   Also, add your broker to the supervision hierarchy in your <%= @app %>.ex:
 
+  Elixir v1.5 or above:
+
+      def start(_type, _args) do
+        children = [
+          # ...
+          {<%= @module %>, []}
+        ]
+
+        supervise(children, strategy: :one_for_one)
```

Comment:
For the new style you need to use `Supervisor.init`. `supervise` is also from `Supervisor.Spec`.

Reply:
Ops, I forgot to change the code after the child spec. Fixed.